spec

```diff
@@ -718,7 +718,6 @@ endseg
 beginseg
     name "ovl_Bg_Gnd_Darkmeiro"
     include "build/src/overlays/actors/ovl_Bg_Gnd_Darkmeiro/z_bg_gnd_darkmeiro.o"
-    include "build/data/overlays/actors/z_bg_gnd_darkmeiro.data.o"
     include "build/data/overlays/actors/z_bg_gnd_darkmeiro.reloc.o"
```

Comment:
Once you get an overlay to fully match you can delete the old reloc file (`data/overlays/actors/z_bg_gnd_darkmeiro.reloc.s` in your case) and use the one auto generated.
```suggestion
    include "build/src/overlays/actors/ovl_Bg_Gnd_Darkmeiro/ovl_Bg_Gnd_Darkmeiro_reloc.o"
```